stories/tab/tab.stories.tsx

```diff
@@ -182,3 +182,62 @@ export const ControlledMode: StoryObj<Component> = {
         );
     },
 };
+
+export const FullWidthBottomBorder: StoryObj<Component> = {
+    render: () => {
+        const [currentIndex, setCurrentIndex] = useState(0);
+
+        const handleTabClick = (_, order) => {
+            setCurrentIndex(order);
+        };
+
+        return (
+            <Tab
+                onTabClick={handleTabClick}
+                currentActive={currentIndex}
+                fullWidthBottomBorder
+            >
```

Comment:
for simplicity, this example can be uncontrolled

```suggestion
            <Tab fullWidthBottomBorder>
```